### PR TITLE
[DEN-2020] Initial add of the Denver 2020 event.

### PR DESCRIPTION
**Pull request by tavisto**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: content/events/2020-denver/location.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
++++
+Title = "Location"
+Type = "event"
+Description = "Location for devopsdays denver 2020"
++++
+
+<div style="text-align: right; float: right;"><img src="https://www.milehighstation.com/wp-content/uploads/2018/09/5b3bb00fca66ad6e370660f0_MHS-Brand-Awareness-Instagram-1.jpg" height="450px"/></div>
```
Comment (bridgetkromhout) [inline content/events/2020-denver/location.md]: We don't recommend embedding images from other sites, as they could remove it or change the hotlinking access at any time; it's more future-proof to put a copy of any image you want to use in  `static/events/2020-denver/`.

**Comment by bridgetkromhout** [comment]
Slack and email list update: removed Evan, added Jason.

File: content/events/2020-denver/location.md (hunk 1 above)
Comment (tavisto) [reply content/events/2020-denver/location.md]: Ah, I didn't even see that, I just copied last years page. I can go scrape their site and fix the image on both years.

File: data/events/2020-denver.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+event_twitter: "devopsdaysrox"  # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "DevOpsDays Denver Coming Up!"  # Edit this to suit your preferences
+friendly: "2020-denver"  # Four digit year and the city name in lower-case. Don't forget the dash!
+ga_tracking_id: "UA-146926657-1"  # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+startdate: 2020-04-26T00:00:00-06:00  # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2020-04-28T23:59:59-06:00  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Custom reusables
+dinner_date: &speaker_dinner  2020-04-26
```
Comment (mattstratton) [inline data/events/2020-denver.yml]: These are interesting. How are you using them? Inquiring minds want to know :)

**Pull request merged**